TrajectoryVisualization traj_vis

---

std_msgs/Bool success